doc/src/development/testenv/theo-setup-vm-windows.md

```diff
@@ -31,21 +31,33 @@ You may also install the required software manually.
   ssh-keygen -t rsa -b 4096 -C "opendut-vm" -f ~/.ssh/id_rsa
   ```
 
+> **Hint**  
```

Comment:
What about navigating to the repository's directory and executing `git config --local ...` instead of `--global`? Than the configuration is only applied to the repository.

Reply:
Using auto.crlf=false seems to be the least invasive option there is. I don't get why this is not the default behavior anyway. Chocolatey recommends installing with this options enabled by default. The install command above '/NoAutoCrlf' does exactly that. For a more elaborate explanation take a look at this post here: https://stackoverflow.com/a/20653073

I see your point in not changing the global values - another approach could be to use gitattributes to set this when checking out the repository. Checkout the following links:
* https://stackoverflow.com/a/27370303
* https://stackoverflow.com/a/40821931

This looks the most promising:
```shell
echo "* -crlf" > .gitattributes
```

Reply:
If there is only a few files types affected by this causing build issues, why not limiting "-crlf" to those types only?